### PR TITLE
Remove trailing slash from static-route to avoid 301 response from In…

### LINKED ISSUE #47: Do not append slash to route when requesting /api/v1/static
Just came across this. The `Models/Statics.php` file appends a trailing slash at the end of the route which causes the response to be a `301 Moved Permanently` and tells that the document has moved to `/api/v1/static?`

Works after the following adjustment:

```diff
    public static function all()
    {
-        $url = static::getRoute() . '/';
+        $url = static::getRoute();
        
        return static::sendRequest($url, false, 'GET', true);
    }

### PR DESCRIPTION
**Pull request by d00p**
…voiceNinja; add raw-flag to all() function to return clean JSON response if called directly but return raw result for get()-wrapper-function to remain functionality, fixes #47

### CODE REVIEW

File: src/InvoiceNinja/Models/Statics.php [hunk 1]
```diff
@@ -6,16 +6,16 @@ class Statics extends AbstractModel
 {
     public static $route = 'static';
 
-    public static function all()
+    public static function all($raw = false)
```
Comment (hillelcoren) [inline src/InvoiceNinja/Models/Statics.php]: I think this needs to be changed to true otherwise it would change the behavior

Comment (d00p) [reply src/InvoiceNinja/Models/Statics.php]: But the expected behaviour is that a correct JSON response is returned when `all()` is called directly. Like any other models.

Comment (hillelcoren) [reply src/InvoiceNinja/Models/Statics.php]: Ok, works for me

**Comment by hillelcoren** [comment]
Thanks!

**Pull request merged**